ui/components/multichain/pages/send/components/quote-card/hooks/useEthFeeData.tsx

```diff
@@ -31,12 +31,15 @@ export default function useEthFeeData(gasLimit = 0) {
   const networkAndAccountSupports1559 = useSelector(
     checkNetworkAndAccountSupports1559,
   );
-  const { medium } = useSelector(getGasFeeEstimates);
+  const { medium, gasPrice: maybeGasFee } = useSelector(getGasFeeEstimates);
+
+  const gasFee1559 = maybeGasFee ?? medium;
 
```

Comment:
Seeing a bug where `ethGasFee` is calculated below. If networkAndAccountSupports1559=false and maybeGasFee is undefined, `medium` is converted into a BigNumber, which throws an exception.

This fixes it locally for me
```suggestion
  const gasFee1559 = maybeGasFee ?? medium?.suggestedMaxFeePerGas;
```

Reply:
![Screenshot 2024-05-10 at 2 52 40 PM](https://github.com/MetaMask/metamask-extension/assets/100321200/d71ca12d-7e72-44d7-905e-40a4caabf8e1)

Reply:
Updated